lib/membrane_timescaledb_reporter/models/model.ex

```diff
@@ -0,0 +1,59 @@
+defmodule Membrane.Telemetry.TimescaleDB.Model do
+  require Logger
+  import Ecto.Query
+  alias Membrane.Telemetry.TimescaleDB.Repo
+  alias Membrane.Telemetry.TimescaleDB.Model.{Metric, ElementPath}
+
+  def create_all_metrics(metrics) do
```

Comment:
This is not a good name for this function. It creates metrics only when they don't exist, but main function is to insert new values for the metric